XMLHTTPRequest/XMLHTTPRequest.m

```diff
@@ -66,12 +69,17 @@ - (void)send:(id)data {
 
     __block __weak XMLHttpRequest *weakSelf = self;
 
-    id completionHandler = ^(NSData *receivedData, NSURLResponse *response, NSError *error) {
-        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
+    id completionHandler = ^(NSData *receivedData, NSURLResponse *resp, NSError *error) {
```

Comment:
Do you mind renaming this back to `response`? I find it more readable than using an abbreviation.